infrastructure/game-server.tf

```diff
@@ -39,3 +39,21 @@ resource "google_secret_manager_secret_iam_binding" "cloud_build_binding" {
     "serviceAccount:cloudbuild-cicd@${var.project}.iam.gserviceaccount.com",
   ]
 }
+
+# Make game cloudbuild.yaml
+resource "local_file" "game-cloudbuild" {
+  content = templatefile(
+    "${path.module}/files/game/cloudbuild.yaml.tpl", {
+      CLIENT_BUCKET = google_storage_bucket.game-client-binaries.name
+  })
+  filename = "${path.module}/../game/cloudbuild.yaml"
+}
+
+resource "google_storage_bucket" "game-client-binaries" {
+  project       = var.project
+  name          = "${var.project}-game-client-binaries"
```

Comment:
Since Cloud Build knows the env var of `${PROJECT_ID}` I think we don't need to generate the cloudbuild.yaml, since we can define it by convention all the way through.

Saves us the complexity of generating yet another file.